components/mod-event/EventList.tsx

```diff
@@ -375,7 +375,7 @@ const EventFilterPanel = ({
               type="datetime-local"
               id="createdAfter"
               name="createdAfter"
-              className="block w-full"
+              className="block w-full dark:[color-scheme:dark]"
```

Comment:
The date picker, by default, adds a calendar icon in the input field. This classname ensures that the icon's color changes based on the browser's scheme.